src/lib.rs

```diff
@@ -204,11 +204,14 @@ mod codegen;
 mod parser;
 mod value;
 mod error;
+mod util;
 
 pub mod short;
 pub mod object;
+pub mod number;
 
 use short::Short;
+use number::Number;
```

Comment:
Yeah, analog to `Short` and `Object`, I'd prefer to "hide" the inner workings of the library. If you do - for whatever reason - want to use those as opposed to actual `JsonValue`, you do so explicitly.

I want to implement the arithmetic `ops` to the `Number` in future releases, re-exporting it at the top level then might make more sense.